lib/bots/versionbot.ts

```diff
@@ -84,12 +84,24 @@ interface FSError {
     message: string;
 }
 
+interface VersionBotConfiguration extends GithubBotConfiguration {
+    githubbot: {
+        versionbot: {
+            minimum_approvals?: number;
+            approved_reviewers?: string[];
+            maintainers?: string[];
+        };
+    };
+}
+
 // The VersionBot is built on top of GithubBot, which does all the heavy lifting and scheduling.
 // It is designed to check for valid `versionist` commit semantics and alter (or merge) a PR
 // accordingly.
 export class VersionBot extends GithubBot.GithubBot {
     // Flowdock Adapter
     private flowdock: FlowdockAdapter;
+    private mergeLabel = 'procbots/versionbot/ready-to-merge';
+    private ignoreLabel = 'procbots/versionbot/no-checks';
```

Comment:
I think you're right, and not just for `this`. It also means all predefined `const`s for the class are at the top of source, so it's easy to find them. I'll move them out.